modules/reitit-middleware/src/reitit/ring/middleware/session.clj

```diff
@@ -0,0 +1,33 @@
+(ns reitit.ring.middleware.session
+  (:require
+   [clojure.spec.alpha :as s]
+   [ring.middleware.session :as session]
+   [ring.middleware.session.memory :as memory]))
+
+(s/def ::spec (s/keys :opt-un [::session]))
```

Comment:
Didn't review this in detail, but the spec `::session` is not defined anywhere.

Reply:
Thank you for noticing it. I actually didn't know how the `:spec` key is used for reitit middlewares and I was trying to follow along https://github.com/metosin/reitit/blob/master/modules/reitit-middleware/src/reitit/ring/middleware/muuntaja.clj#L6-L7.
In the next commit I'll indicate the `:session` is a map of https://github.com/ring-clojure/ring/blob/master/ring-core/src/ring/middleware/session.clj#L85-L100